src/java.base/unix/classes/sun/nio/fs/UnixFileAttributes.java

```diff
@@ -58,6 +58,7 @@ class UnixFileAttributes
     private long    st_ctime_nsec;
     private long    st_birthtime_sec;
     private long    st_birthtime_nsec;
+    private boolean birthtime_available = true;
```

Comment:
I have a mild preference to leave the default be false, then set it to true in the native code when setting the birth time.